File: BEGINNER/1182 Coluna na Matriz.py
# SUBPROGRAMA
def fazerMatriz():
	minhaMatriz = []
	for col in range(12):
		linha = []
		for lin in range(12):
			linha.append(float(input()))
		minhaMatriz.append(linha)
	return minhaMatriz


def somarColuna(colunaOp, matriz):
	somaDaColuna = 0
	for lin in range(len(matriz)):
		somaDaColuna += matriz[lin][colunaOp]
	return somaDaColuna


def mediarColuna(ColunaOp, matriz):
	somaColuna = somarColuna(ColunaOp, matriz)
	mediaDaColuna = somaColuna / len(matriz)
	return mediaDaColuna


# PRINCIPAL
colunaDaOperacao = int(input())
operacao = input()
essaMatriz = fazerMatriz()
if operacao == 'S':
	soma = somarColuna(colunaDaOperacao, essaMatriz)
	print('%.1f' % soma)
else:
	media = mediarColuna(colunaDaOperacao, essaMatriz)
	print('%.1f' % media)

File: C/INICIANTE/1182 Coluna na Matriz.c
#include <stdio.h>
int main()
{
    int colunaDaOperacao, i, j;
    char operacao[2];
    double matriz[12][12], soma;
    soma = 0;
    scanf("%d %s", &colunaDaOperacao, operacao);
    for (i=0; i<12; i++)
        for (j=0; j<12; j++)
            scanf("%lf", &matriz[i][j]);
    for (i=0; i<12; i++)
        for (j=0; j<12; j++)
            if (j == colunaDaOperacao)
                soma += matriz[i][j];
    if (operacao[0] == 'S')
        printf("%.1lf\n", soma);
    else
        printf("%.1lf\n", (soma/12.0));
    return 0;
}
